tests/console/supportutils.pm

```diff
@@ -23,8 +23,9 @@ sub run {
     my $self = shift;
     $self->select_serial_terminal;
 
+    my $options = get_var('SUPPORTCOFIG_OPTIONS') // 'BOOT,BTRFS,DAEMONS,CRASH,MEM,NET,PROC,SYSCONFIG';
```

Comment:
get_var does have integrated default value. https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm#L751

Reply:
Thx

Reply:
From the verify log without SUPPORTCOFIG_OPTIONS set, supportconfig run with expected default options. http://openqa.nue.suse.com/tests/5140027 But OSD is down now.
Any suggestion here?